.github/workflows/ci.yml

```diff
@@ -55,6 +55,7 @@ jobs:
   # We cannot run the full CI in devcontainers, as we will run out of disk space in GitHub Actions.
   # Instead, we just validate the devcontainer is built correctly by running a minimal check below.
   validate-devcontainer:
+    needs: "ci"
```

Comment:
Is this still needed? I tested CI without it, and it is passing ([run](https://github.com/OmarTawfik-forks/slang/actions/runs/13449971093)).

Reply:
re: your earlier comment about multiple jobs in the same workflow having the same disk size limitation:
I don't think both jobs are necessarily executed on the same GitHub-hosted runner. They are assigned at random based on availability in Azure.
The sanctuary workflow is build/running Slang in 32 concurrent jobs without any failures.